Automattic/wp-calypso
# Update Splash Screen

#### Changes proposed in this Pull Request

This probably updates the splash screen? (I confess, not the most confident PR ever)

It uses the HEX code for Classic Bright/Blue, similar to what the theme colour does. The flash of colour is an issue beyond just the splash screen.

#### Testing instructions

In all honesty, I can't manage to test this myself but Google seems to suggest that this is the change to make: https://developers.google.com/web/fundamentals/web-app-manifest/#splash-screen

You can navigate to the Application tab of dev tools to Manifest to see the different background colour and the new white logos.

<img width="439" alt="Screenshot 2019-05-05 at 21 52 22" src="https://user-images.githubusercontent.com/43215253/57200169-19bfad80-6f80-11e9-9694-38da4012f141.png">

cc @blowery, @jblz 

Fixes #29685
